### PR TITLE
Re-write backend in Node

### PR DESCRIPTION
**Pull request by iafisher**
Fixes: #3

Disclaimer: I'm an engineer at Google

### CODE REVIEW

File: backend/models/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+'use strict';
```
Comment (iafisher) [inline backend/models/index.js:1]: This file was auto-created by Sequelize.

File: backend/migrations/20200409160238-create-project.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
```
Comment (noahdkim) [inline backend/migrations/20200409160238-create-project.js:33]: what is this?

File: backend/models/index.js (hunk 1 above)
Comment (noahdkim) [reply backend/models/index.js:1]: ok i'm just assuming this is good

File: backend/views/vanilla.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
-{% load static %}
 <!DOCTYPE html>
 
 <html lang="en">
 
 <head>
-    <link rel="stylesheet" href="{% static 'ui/global.css' %}">
-    <link rel="stylesheet" href="{% static style %}">
-    <script defer src="{% static script %}"></script>
+    <title>[vanilla] pm</title>
```
Comment (noahdkim) [inline backend/views/vanilla.html:6]: what's pm mean?

Comment (iafisher) [reply backend/views/vanilla.html:6]: project management

File: backend/migrations/20200409160238-create-project.js (hunk 2 above)
Comment (iafisher) [reply backend/migrations/20200409160238-create-project.js:33]: No idea, sequelize generated this code for me.

**Pull request merged**